docs/admin/templates/extending-templates/workspace-tags.md

```diff
@@ -66,6 +70,14 @@ the workspace owner to change a provisioner group (due to different tags). In
 most cases, `coder_parameter`s backing `coder_workspace_tags` should be marked
 as immutable and set only once, during workspace creation.
 
+**We recommend only using the following as inputs for `coder_workspace_tags`:**
+
+- Static values (e.g. `"developers"`),
+- Template variables (e.g. `var.az`),
+- Coder parameters (e.g. `data.coder_parameter.runtime_selector.value`).
```

Comment:
instead of using the `e.g.`, how does this seem?

```suggestion
|                    | Example |
|:-------------------|:--------|
| Static values      | `developers` |
| Template variables | `var.az` |
| Coder parameters   | `data.coder_parameter.runtime_selector.value` |
```

which looks like this:

|                    | Example |
|:-------------------|:--------|
| Static values      | `developers` |
| Template variables | `var.az` |
| Coder parameters   | `data.coder_parameter.runtime_selector.value` |

<https://developers.google.com/style/abbreviations#dont-use>